src/abstractions/software/assembler/code/index.js

```diff
@@ -1,5 +1,7 @@
+import { zeroPad, convertToBinary } from './utils'
+
 const DESTIONATION_CODES = Object.freeze({
-  NULL: '000',
+  '': '000',
```

Comment:
since `NULL` and '' are not technically destination codes, they should be removed. Their should be a higher level check that handles their role.

Reply:
I thought, machine codes should be found at one place instead of returning `000` in the handler. But we can do that.